### DISCUSSION TITLE
Document Title bug when bulk importing from .docx

### DISCUSSION THREAD

**Discussion by adriansoewono** [outline/outline#8089] [General]
Hi, it seems to me the Document Title is buggy when I tried to bulk import from .docx files which has a dot character in the filename.

so to reproduce:

1. all my current document filename structure has a number+dot prefix (so I can set the document order easily).
I have to use .docx files because I need to import images easily.
![Screenshot 1](https://github.com/user-attachments/assets/b26f3304-6208-49da-8a6a-8843ddd881ac)

2. I bulk import it using files drag and drop
![Screenshot 2](https://github.com/user-attachments/assets/59ce8bbb-345c-4319-b16f-db26c04b29bc)

3. then the document title in the Outline shows like this, there's a backslash character before the dot.
![Screenshot 3](https://github.com/user-attachments/assets/85a87f53-f977-4c3d-b2ed-57d5b3c37fb2)

apparently this happens only in .docx import,
I have tried to import .md files, the document title works without any backslash problem for the dot character.

now it seems like a simple bug,
but since all my documents are using number and dot prefix in the filename,
this problem makes me have to rename all my documents in the Outline after importing, which is really a tremendous task.

it will be great if Outline can fix the backslash character bug.

**Comment by tommoor** [comment]
Converted to bug here – https://github.com/outline/outline/issues/8090

One note is that the file name should only be used as the title (fallback) if there isn't one in the document itself.
